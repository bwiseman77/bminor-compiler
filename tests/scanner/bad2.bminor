int_
$BashVariable
double .0002
