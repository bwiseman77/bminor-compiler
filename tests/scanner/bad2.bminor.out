-------------------------------------------
           token type  text
-------------------------------------------
               IDENT  int_
          SCAN ERROR  $
               IDENT  BashVariable
               IDENT  double
          SCAN ERROR  .
     INTEGER_LITERAL  0002
